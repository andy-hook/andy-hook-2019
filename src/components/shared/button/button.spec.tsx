import React from "react"
import renderer from "react-test-renderer"

import Button from "./button"

describe("<Logo />", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Button href="https://www.google.com" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
