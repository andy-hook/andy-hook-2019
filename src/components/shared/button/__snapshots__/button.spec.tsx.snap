// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders correctly 1`] = `
<a
  className="sc-bdVaJa kYVele"
  href="https://www.google.com"
  onClick={[Function]}
  target="_blank"
>
  <span
    className="sc-bwzfXH hGubuC"
  >
    This is a button
  </span>
</a>
`;
